### PR TITLE
Refactoring branch

### PR DESCRIPTION
**Pull request by RLudchenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
 "# hibernate-cinema"
```
Comment (RomanDubovskyi) [inline README.md]:
```suggestion
# hibernate-cinema
```

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
 "# hibernate-cinema" 
+# Project Description 
+This project represents small online cinema which has functions like: 
+Registration, Login, Add Product To Cart, Complete Order, etc. 
+Users can also read a description of their orders.
```
Comment (RomanDubovskyi) [inline README.md]:
```suggestion
This project represents small online cinema which has functions like: 
register account, Login, book ticket, etc. 
Users can also read a description of their orders. 
```

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+# Build Pattern 
+This project uses MVC pattern, it is structured using Dao and Service 
+layers which make it easier to read for other developers.
+- **_DAO_** layer contains basic CRUD operations with the persistence layer
+- **_Service_** layer contains business logic for the application  
+- **_Servlets_**, implement client-server communicating logic   
+
+# DBMS 
+The project closely works with SQL queries and provides options to store data 
+in comfortable tables, each table has its own purpose.
```
Comment (RomanDubovskyi) [inline README.md]: it would be great to add instruction how to run your project

**Review by RomanDubovskyi — COMMENTED** [review]
travis failed

File: README.md (hunk 2 above)
Comment (RLudchenko) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (RLudchenko) [reply README.md]: Done :)

File: README.md (hunk 1 above)
Comment (RLudchenko) [reply README.md]: Done

**Comment by RLudchenko** [comment]
@RomanDubovskyi build successful у Maven, не знаю чому валиться Travis

**Pull request merged**